llvm/lib/ProfileData/InstrProfWriter.cpp

```diff
@@ -190,7 +199,16 @@ InstrProfWriter::InstrProfWriter(
       InfoObj(new InstrProfRecordWriterTrait()),
       WritePrevVersion(WritePrevVersion),
       MemProfVersionRequested(MemProfVersionRequested),
-      MemProfFullSchema(MemProfFullSchema) {}
+      MemProfFullSchema(MemProfFullSchema) {
```

Comment:
For internal tooling it will be cleaner if we plumb it through as a parameter instead of setting the cl::opt.

Reply:
done